prow/github/helpers_test.go

```diff
@@ -131,3 +131,60 @@ func TestPermissionsFromLevel(t *testing.T) {
 		}
 	}
 }
+
+func TestMatchCardToContent(t *testing.T) {
+	var testCases = []struct {
+		card       ProjectCard
```

Comment:
This only needs to be a string, right?

Reply:
Just for readability, I am using a card that contains the string. It can be confusing which string means which. I tried keep the card contents pristine. Except code plugins have currently does not use the object URL consistently and this the reason I want to handle both url and html_url. We cannot handle issue or PR number as that cannot be unique in a org.

Reply:
I think it's more confusing to have extra data in the test that is not used for anything. It's up to the test to use the correct field in the card in the call to the underlying func, etc